web/server/codechecker_server/api/mass_store_run.py

```diff
@@ -506,23 +541,26 @@ def __store_source_files(
                 continue
 
             with DBSession(self.__Session) as session:
-                file_path_to_id[trimmed_file_path] = self.__add_file_content(
-                    session, trimmed_file_path, source_file_name, file_hash)
+                self.__add_file_content(
+                    session, source_file_name, file_hash, blame_info)
+
+                file_path_to_id[trimmed_file_path] = add_file_record(
+                    session, trimmed_file_path, file_hash, remote_url,
+                    tracking_branch)
 
         return file_path_to_id
 
     def __add_file_content(
         self,
         session: DBSession,
-        file_path: str,
         source_file_name: str,
-        content_hash: str
-    ) -> int:
+        content_hash: str,
+        blame_info: Optional[dict]
+    ):
         """
-        Add the necessary file contents. If the file is already stored in the
-        database then its ID returns. If content_hash in None then this
-        function calculates the content hash. Or if is available at the caller
-        and is provided then it will not be calculated again.
+        Add the necessary file contents. If content_hash in None then this
```

Comment:
According to this documentation `content_hash` can be none, however, type hint says `str`. Maybe it should be `Optional[str]`.